dataplane/saiserver/acl.go

```diff
@@ -292,3 +292,118 @@ func (a *acl) CreateAclEntry(ctx context.Context, req *saipb.CreateAclEntryReque
 
 	return &saipb.CreateAclEntryResponse{Oid: id}, nil
 }
+
+type myMacInfo struct {
+	priority       *uint32
+	portID         *uint64 // wildcard if not specified
+	vlanID         *uint32 // wildcard if not specified
```

Comment:
make this uint16, its 32 bits because protobuf hasn't have uint16 types

Reply:
Revised the code.